taipy/core/_repository/_filesystem_repository.py

```diff
@@ -123,17 +123,11 @@ def _export(self, entity_id: str, folder_path: Union[str, pathlib.Path]):
         else:
             folder = folder_path
 
-        if folder.resolve() == self._storage_folder.resolve():
-            raise InvalidExportPath("The export folder must not be the storage folder.")
-
         export_dir = folder / self._dir_name
         if not export_dir.exists():
             export_dir.mkdir(parents=True)
 
         export_path = export_dir / f"{entity_id}.json"
-        # Delete if exists.
-        if export_path.exists():
-            export_path.unlink()
```

Comment:
@toan-quach asked:
> why are we removing this code?

It's is already handled in the `taipy.import()` function, so this is duplicated.